### DISCUSSION TITLE
Stereo Audio

### DISCUSSION THREAD

**Discussion by The30Note** [bepvte/bd-addons#20] [Q&A] [answered]
Hey, I understand the stereo audio plugin has been broken by the new better discord client. Are there any intentions on fixing this or finding a work around, I'd be willing to help if at all possible thanks.

**Comment by bepvte** [comment, marked as answer]
It was broken by discord. I will not be fixing it, because its very hard and I don't want to.

**Comment by jjbendbloxks** [comment]
yo bep i just wanna say i use your plugins and i was wondering if you could add bitrate to the stereo plugin if thats possible maybe??

**Reply by bepvte** [reply]
Arent you the guy who bothers me on discord to add bitrate? What makes you think I'd change my mind? And this plugin is broken, I don't fix it anymore.

**Reply by jjbendbloxks** [reply]
? no this is the first time and aight bruh thats all u had to say mb for bothering

**Comment by The30Note** [comment]
Why am I cced on this lol

**Reply by bepvte** [reply]
![image](https://user-images.githubusercontent.com/8226605/139337780-b4b3779f-5933-4d6f-9347-a30cc885f323.png)
